cachi2/core/package_managers/yarn/main.py

```diff
@@ -121,6 +121,7 @@ def _set_yarnrc_configuration(project: Project, output_dir: RootedPath) -> None:
     yarn_rc.enable_immutable_installs = True
     yarn_rc.enable_strict_ssl = True
     yarn_rc.enable_telemetry = False
+    yarn_rc.ignore_path = False
```

Comment:
Nitpick: Should we add to commit message, which option is related to this code change? Something like: In case the user sets ignorePath option ...   

So it's clear which option needs to be set without checking the code?

Reply:
The config option is decribed in the commit details:

```
Sets the ignorePath option to true, which makes yarn ignore the
executable defined yarnPath and fallback to the global one. This makes
Corepack fetch a new yarn executable, which will then be used.
```

Did you mean adding it to the main commit message?

Reply:
No that's enough. I apologize if I missed it originally. Thanks